contracts/BatchReveal.sol

```diff
@@ -1,11 +1,10 @@
 pragma solidity ^0.8.2;
 
 abstract contract BatchReveal {
-    uint constant public TOKEN_LIMIT = 9900;
-    uint constant public REVEAL_BATCH_SIZE = 900;
-    uint constant public CONTRIBUTOR_OFFSET = 100;
-    mapping(uint => uint) public batchToSeed;
-    uint public lastTokenRevealed = 0; // in [0-9900]. offset not included
+    uint256 public constant TOKEN_LIMIT = 10000;
+    uint256 public constant REVEAL_BATCH_SIZE = 1000;
+    mapping(uint256 => uint256) public batchToSeed;
+    uint256 public lastTokenRevealed = 0; // in [0-9999]. offset not included
```

Comment:
```suggestion
    uint256 public lastTokenRevealed = 0; // in [0-9999]
```

does this also need to be updated to start with 1? not sure

Reply:
its awkward and I think it needs some refinement but it does work